assignments/array-methods.js

```diff
@@ -55,23 +55,37 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
-let fullName = [];
+
+let fullName = new Array();
+runners.forEach(function(element){
+  fullName.push(element.first_name + " " + element.last_name);
+})
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
-console.log(allCaps); 
+
+const allCaps = runners.map(function(currentValue){
+    return currentValue.first_name.toUpperCase();
+  });
+  console.log(allCaps); 
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue.  Get a list of runners with large sized shirts so they can choose a different size. Return an array named largeShirts that contains information about the runners that have a shirt size of L and log the result
-let largeShirts = [];
-console.log(largeShirts);
+
+const largeShirts = runners.filter(function(currentValue){
+    return currentValue.shirt_size == "L";
+  });
+  console.log(largeShirts);  
```

Comment:
I'd recommend using a **strict** equality operator here (`===`) over the **abstract** equality operator (`==`) since the latter will use type coercion (and in cases where the values have the same type, slow it down).